deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/OafResource.java

```diff
@@ -94,8 +93,14 @@ public class OafResource implements Resource {
     private OafDatasetConfiguration oafConfiguration;
 
     private HtmlViewConfiguration htmlViewConfiguration;
-
-    public OafResource( ResourceMetadata<Resource> metadata, Workspace workspace, DeegreeOAF config ) {
+    
+    private static Map<String, List<ConfigureCollection>> additionalCollectionMap = new HashMap<>();
```

Comment:
Isn't `OafResource` representing one dataset?
A `static` config here does not make sense for me, I would expect the configuration for exactly that dataset here.

Reply:
Yes, correct. I have corrected it now.